walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,78 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.HashSet;
 
+/** Finds options for times to schedule a meeting request */
```

Comment:
It'd be great to document what the return value means. Is it a set of acceptable *start* times for the meeting? Or a list of time ranges in which the meeting would fit (where the user is responsible for choosing a valid start time)? Or something else?

Also good to document that "events" is the universe of all previously scheduled events, possibly including events that nobody in MeetingRequest.getAttendees() is attending.

Reply:
Is this a better description? 

Finds options for times to schedule a meeting request.
Given all events that occur in the day and a meeting request with attendees
that may or may not have conflicting events, returns a list of time ranges
during which the requested meeting could occur.

Reply:
That's better. The return value is still a little unclear though. My best attempt is:

Returns a set of non-overlapping time intervals of duration at least `request.getDuration()`, during which all attendees are available.

(Is that correct though?)

Reply:
Yep, that's correct. I changed it to 

Finds options for times to schedule a meeting request.
  Given all events that occur in the day and a meeting request with attendees
  that may or may not have conflicting events, returns a set of non-overlapping time intervals 
  of duration at least the requested meeting length, during which all attendees are available.